ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolConfiguration.java

```diff
@@ -87,7 +87,7 @@ enum Implementation {
   long DEFAULT_PENDING_TRANSACTIONS_LAYER_MAX_CAPACITY_BYTES = 25_000_000L;
   int DEFAULT_MAX_PRIORITIZED_TRANSACTIONS = 4000;
   EnumMap<TransactionType, Integer> DEFAULT_MAX_PRIORITIZED_TRANSACTIONS_BY_TYPE =
-      new EnumMap<>(Map.of(TransactionType.BLOB, 9));
+      new EnumMap<>(Map.of(TransactionType.BLOB, 20));
```

Comment:
Why 20?

Reply:
That is the max being tested on fusaka-devnet-0

Reply:
That's fine, just wanted to know where the number came from